apps/next-evans/styles/globals.css

```diff
@@ -37,4 +37,32 @@
   border-bottom: 1px dotted currentColor;
 }
 
+.bracketed::before {
+  content: '[';
+  padding-right: 0.4em;
+}
+
+.bracketed::after {
+  content: ']';
+  padding-left: 0.4em;
+}
+
+.diamonds li:before {
```

Comment:
are all of these diamond related classes for your _other_ PR?

Reply:
No, they're for the `<ul>` tags in the mdx pages, you have the bullets as diamonds